distribution/docker/docker-test-entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+cd /usr/share/elasticsearch/bin/
+./elasticsearch-users useradd x_pack_rest_user -p x-pack-test-password -r superuser || true 
```

Comment:
Can we do these with docker exec instead of modifying the entry point?

Reply:
I would rather not. The test fixtures are transparent w.r.t the technology used to implement them. In general there's nothing docker specific in the build scripts, and the plugin disables everything that can't run if docker is not available. I would like to keep this and thus avoid using docker exec. 

Maybe this is a generic enough use-case to add env vars for creating a super-user when bringing up the container if these are present ?